### ISSUE TITLE
Preferences Startup folders - not working since 0.9.1

### ISSUE THREAD

**Issue by wernegrh** [mucommander/mucommander#110]
At 'Startup folders' selecting 'Custom folder' worked fine up to 0.9.0.
Now the design was modified (one general pair of buttons instead of left and right separate), input is possible, but starts always into the 'Last visited folder'.
Pretty much annoying, if you quit 'mucommander' with a pane on a drive, which is no longer available at next start (i.e. NAS is down).
I saw 'preferences.xml' shows a slightly modified structure, maybe the main program is not yet aware of this?

**Comment by ahadas** [comment]
Hi @wernegrh ,
Thanks for reporting this.
I fixed it by always loading the custom folders as you requested, although I'm not sure that setting custom folders it is the real fix for this. We can further discuss it if you'd like, I'll be interested to know how the application behaved in that case.

**Comment by wernegrh** [comment]
Hello all,

sorry for my late answer - I was on busyness trip some weeks.

I have not created java projects yet, and not worked with gradle.
I struggled a lot until I realized, it is best to pull the repository 
instead of downloading the ZIP file only.

Now gradle works, but at the end I get more than 11000 unresolved 
dependencies - in other words: nothing resolved:-( I have no idea, what 
is wrong with my java or the libraries.
And, I could not check the version 0.9.3 this way - sorry.

So, I just wait for your release 0.9.3.

Best regards

Gerhard Werner

-- 
Mit freundlichen Grüßen

Gerhard Werner
Untere Kreuzgasse 5
90403 Nürnberg

Tel. : +49 911 236929-40
FAX  : +49 911 236929-45
Mobil: +49 1577 6083856
